### PR TITLE
Milkytracker version bump

### PR DESCRIPTION
**Pull request by extrowerk**
No idea what happend with the "r4"s.

### CODE REVIEW

**Comment by extrowerk** [comment]
@scottmc : patch already upstreamed and merged.
I got no gcc2 Haiku to test it.

File: media-sound/milkytracker/patches/milkytracker-1.01.00.patchset [hunk 1]
```diff
@@ -36,7 +36,7 @@ index 39e0412..2931ecf 100644
  SubDirHdrs - ;
  
 -LINKLIBS on MilkyTracker = -lbe -lgame -lmedia -lmidi2 -ltextencoding -ltracker ;
-+LINKLIBS on MilkyTracker = -lbe -lgame -lmedia -lmidi2 -ltextencoding -ltracker -lstdc++.r4 ;
```
Comment (korli) [inline media-sound/milkytracker/patches/milkytracker-1.01.00.patchset]: Why is this needed when you only tested on x86_64?

File: media-sound/milkytracker/patches/milkytracker-1.01.00.patchset [hunk 2]
```diff
@@ -48,15 +48,15 @@ index 662ee18..b76c586 100644
  SubDir TOP src tracker haiku MilkySettings ;
  
 -LINKLIBS on MilkySettings = -lbe -lmidi2 ;
-+LINKLIBS on MilkySettings = -lbe -lmidi2 -lstdc++.r4 ;
```
Comment (korli) [inline media-sound/milkytracker/patches/milkytracker-1.01.00.patchset]: Ditto.

File: media-sound/milkytracker/patches/milkytracker-1.01.00.patchset [hunk 3]
```diff
@@ -137,8 +137,8 @@ index 2931ecf..a9dc7db 100644
  SubDirHdrs $(PathTracker) haiku ;
  SubDirHdrs - ;
  
--LINKLIBS on MilkyTracker = -lbe -lgame -lmedia -lmidi2 -ltextencoding -ltracker -lstdc++.r4 ;
```
Comment (korli) [inline media-sound/milkytracker/patches/milkytracker-1.01.00.patchset]: Ditto

File: media-sound/milkytracker/patches/milkytracker-1.01.00.patchset [hunk 4]
```diff
@@ -166,8 +166,8 @@ index b76c586..4ece810 100644
 @@ -1,6 +1,6 @@
  SubDir TOP src tracker haiku MilkySettings ;
  
--LINKLIBS on MilkySettings = -lbe -lmidi2 -lstdc++.r4 ;
```
Comment (korli) [inline media-sound/milkytracker/patches/milkytracker-1.01.00.patchset]: Ditto

File: media-sound/milkytracker/patches/milkytracker-1.01.00.patchset (hunk 1 above)
Comment (extrowerk) [reply media-sound/milkytracker/patches/milkytracker-1.01.00.patchset]: Nah, i didn't changed anything here, just applied the patchset, added the missing file, extracted the patches, and it got changed. Why: it is unclear to me. I can however change them back.

**Comment by extrowerk** [comment]
Afaik the Patch is fixed, but not tested on gcc2. Somebody up to Test it?
